### DISCUSSION TITLE
In children's classic, Winnie-the-Pooh, what type of animal is Eeyore?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#650] [Galaxy Trivia] [answered]
**Bonus:** Can you name all of Pooh's friends?

**Comment by DevElkami** [comment]
Eeyore is a donkey.

Piglet is Pooh's best friend besides Christopher Robin.

**Comment by peterbaccaro** [comment, marked as answer]
Eeyore is an old grey **donkey**.
Pooh's friends are **Piglet**, **Eeyore**, **Kanga**, **Roo**, **Owl**, **Rabbit**, and **Tigger**.
